sky/build/SnapshotterInvoke

```diff
@@ -74,6 +74,8 @@ SnapshotProject() {
 
   local project_path="$1"
 
+  RunCommand pushd ${project_path}
```

Comment:
The entire script needs to be run from the project directory and its a separate xcodebuild phase. A `cd` would have done too.